core/ledger/kvledger/txmgmt/statedb/statecouchdb/couchdb.go

```diff
@@ -1814,12 +1814,16 @@ func (couchInstance *couchInstance) handleRequest(ctx context.Context, method, d
 		// If the maxRetries is greater than 0, then log the retry info
 		if maxRetries > 0 {
 
+			retryMessage := fmt.Sprintf("Retrying couchdb request in %s", waitDuration.String())
```

Comment:
nit: `waitDuration` is already a `fmt.Stringer` so the call to `String()` is unnecessary.

Reply:
done.